functionary/core/models/workflow.py

```diff
@@ -96,3 +96,7 @@ def deactivate(self):
         # TODO: Once scheduled_tasks support workflows come back and do something like:
         # for scheduled_task in self.scheduled_tasks.all():
         #     scheduled_task.pause()
+
+    def is_active(self) -> bool:
```

Comment:
I sort of implied this in another comment, but I would just make this a property method.